modules/helm/crds.mk

```diff
@@ -37,29 +37,31 @@ ifeq ($(HOST_OS),darwin)
 	sed_inplace := sed -i ''
 endif
 
+crds_dir ?= deploy/crds
+crds_dir_readme := $(dir $(lastword $(MAKEFILE_LIST)))/crds_dir.README.md
+
 .PHONY: generate-crds
 ## Generate CRD manifests.
 ## @category [shared] Generate/ Verify
 generate-crds: | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ)
-	$(eval crds_gen_temp := $(bin_dir)/scratch/crds)
 	$(eval directories := $(shell ls -d */ | grep -v -e 'make' $(shell git check-ignore -- * | sed 's/^/-e /')))
 
-	rm -rf $(crds_gen_temp)
-	mkdir -p $(crds_gen_temp)
+	mkdir -p $(crds_dir)
+	cat $(crds_dir_readme) > $(crds_dir)/README.md
```

Comment:
nitpick (non-blocking): I find the use of `cat` here a bit surprising - I'd think `cp` would be more appropriate? (`cat` has to read the file and output it - `cp` doesn't need to do that)

Reply:
Agreed, and should be fixed now.